src/form/en_acr.jl

```diff
@@ -0,0 +1,840 @@
+# GENERATOR
+
+# GENERATOR - Variables
+
+"""
+	function variable_mc_generator_power(
+		pm::AbstractExplicitNeutralACRModel;
+		nw::Int=nw_id_default,
+		kwargs...
+	)
+
+For ACR models with explicit neutrals,
+creates generator power variables `:pg` and `:qg`, 
+and placeholder dictionaries for terminal power flows `:pg_bus` and `:qg_bus`,
+"""
+function variable_mc_generator_power(pm::AbstractExplicitNeutralACRModel; nw::Int=nw_id_default, kwargs...)
+    variable_mc_generator_power_real(pm; nw=nw, kwargs...)
+    variable_mc_generator_power_imaginary(pm; nw=nw, kwargs...)
+    var(pm, nw)[:pg_bus] = Dict{Int, Any}()
+    var(pm, nw)[:qg_bus] = Dict{Int, Any}()
+end
+
+
+# GENERATOR - Constraints
+
+"""
+	function constraint_mc_generator_power_wye(
+		pm::AbstractExplicitNeutralACRModel,
+		nw::Int,
+		id::Int,
+		bus_id::Int,
+		connections::Vector{Int},
+		pmin::Vector{<:Real},
+		pmax::Vector{<:Real},
+		qmin::Vector{<:Real},
+		qmax::Vector{<:Real};
+		report::Bool=true,
+		bounded::Bool=true
+	)
+
+For ACR models with explicit neutrals,
+links the terminal power flows `:pg_bus` and `:qg_bus` to the power variables `:pg` and `:qg` for wye-connected generators
+"""
+function constraint_mc_generator_power_wye(pm::AbstractExplicitNeutralACRModel, nw::Int, id::Int, bus_id::Int, connections::Vector{Int}, pmin::Vector{<:Real}, pmax::Vector{<:Real}, qmin::Vector{<:Real}, qmax::Vector{<:Real}; report::Bool=true, bounded::Bool=true)
+    vr = var(pm, nw, :vr, bus_id)
+    vi = var(pm, nw, :vi, bus_id)
+    pg = var(pm, nw, :pg, id)
+    qg = var(pm, nw, :qg, id)
+
+    phases = connections[1:end-1]
+    n      = connections[end]
+    P      = length(phases)
```

Comment:
what happens if `connections` is not organized by `[phases..., neutral]`? E.g., in cases where we pad properties, i.e. on single phase delta components, the padded properties are added to the end, so `connections` could look like `[1,4,2,3]`. Is there a way we should keep track of which terminal is neutral other than assuming it is the last terminal?

Reply:
See suggestions at the bottom; in my four-wire workflow I turn of the padding with the newly added flag for that.